.travis/README.md

```diff
@@ -0,0 +1,185 @@
+```
+
+This command creates an interactive session that requests your app name, organization name, and builder. You can type in a unique app name or leave it blank if you want Fly to autogenerate a unique name to avoid namespace collisions. Fly will also reserve the app name on the platform, for when you deploy. Select `Dockerfile` as your builder.
+
+By the end of the session, you should have a `fly.toml` file in your app directory that looks similar to the following:
+
+```
+app = "morning-frost-7191"
+
+
+[[services]]
+  internal_port = 8080
+  protocol = "tcp"
+
+  [services.concurrency]
+    hard_limit = 25
+    soft_limit = 20
+
+  [[services.ports]]
+    handlers = ["http"]
+    port = "80"
+
+  [[services.ports]]
+    handlers = ["tls", "http"]
+    port = "443"
+
+  [[services.tcp_checks]]
+    interval = 10000
+    timeout = 2000
+```
+
+### Configuring Travis CI for deployment
+Travis CI uses a [`.travis.yml` file](https://docs.travis-ci.com/user/customizing-the-build) to configure the build, test, and deployment commands used during continuous integration. For this tutorial, you will add testing and deployment instructions to the `.travis.yml` file.
+
+In your Flygreeting application, create a new file called `.travis.yml` and add the following:
+
+```
+language: go
+go:
+    - 1.14
+
+script:
+    - go test -v
+
+deploy: 
+    provider: script
+    script: bash scripts/deploy.sh
+    on:
+        branch: main
+```
+
+This tells Travis CI that our application uses [Go 1.14](https://golang.org/). Travis will use its [Go environment](https://docs.travis-ci.com/user/languages/go/) to run the script, `go test -v` specified. If the tests fail, the whole build fails. Finally, the `deploy` section instructs Travis to run the `scripts/deploy.sh` file if the build passes on the `main` branch.
+
+### Writing the Deployment Script
+Next, create a deploy script to install and run the Flyctl command-line tool on the Travis CI virtual environment after the build succeeds. Create a new directory called `scripts` with a file called `deploy.sh` inside:
+
+```
+mkdir scripts && touch scripts/deploy.sh
+```
+
+Open the newly created `deploy.sh` in an editor and add the following:
+
+```
+#!/bin/sh -l
+
+# install flyctl
+curl -L https://fly.io/install.sh | sh
+
+# set flyctl environmental variables
+export FLYCTL_INSTALL="/home/travis/.fly"
+
+export PATH="$FLYCTL_INSTALL/bin:$PATH"
+
+echo "Successfully Installed Flyctl"
+
+# deploy app
+sh -c "flyctl deploy"
+
+# get app Information
+sh -c "flyctl info"
+
+exit 0
+```
+
+This command installs the Flyctl command-line tool and sets the environmental variables that make it possible to use the shortcut `flyctl` command. Next, it runs `flyctl deploy` to deploy the `flygreeting` application using the `fly.toml` file. Once the application is deployed, it runs `flyctl info` to give you information about the newly deployed app, including the name, version, status, and hostname.
```

Comment:
flyctl is the command, not a shortcut. More accurate to say it adds flyctl to the path  so that it can be run easily.